### DISCUSSION TITLE
[Shipped] Recommended Event in Issue Details

### DISCUSSION THREAD

**Discussion by rachrwang** [getsentry/sentry#54522] [Launch Feedback]
👋 Hi everyone! I'm Rachel, PM on the Issues team.

I wanted to share an update to the Issues experience, inspired by the feedback we received from you in our [quarterly satisfaction surveys](https://github.com/getsentry/sentry/discussions/43531).

Prior to this change, Sentry always showed the most recent event upon navigation into Issue Details, regardless of the contents of that event or how you navigated there. Now, Sentry will display a "recommended event" that looks like this in the UI:

<img width="399" alt="Screenshot 2023-08-09 at 10 18 11 AM" src="https://github.com/getsentry/sentry/assets/82916311/b289f43f-772a-4c87-8692-8778582a5d65">

The "recommended event" is detected by Sentry as **an event from the most recent 7 days that also contains the most context**, which could be a replay, trace, or search terms from the Issues tab. A related update we've made is simplifying navigation across events.

As you use this new experience, we'd love to understand:

1. What are your overall thoughts on this new default?
2. Do you find "recommended" events to be more useful than "latest"? Why or why not?
3. Are there other suggestions you have for making Issue Details more actionable? 

P.S. If you prefer to keep your default to latest event, you can configure your preferences at https://[your team].sentry.io/settings/account/details/. If you do take this action - please let us know why in the comments below

**Comment by estj92** [comment]
> P.S. If you prefer to keep your default to latest event, you can configure your preferences at https://[your team].sentry.io/settings/account/details/. If you do take this action - please let us know why in the comments below

"Latest event" is what I use most, so I have changed it to that.

**Reply by rachrwang** [reply]
@estj92 - thank you for sharing this feedback! Is there anything specific to the "latest event" that you find most valuable? Do you use Replay or Trace View as part of your debugging workflow?

**Comment by ewdicus** [comment]
I've changed my default to "latest event" as it's almost always preferable for debugging. If, for example, the error pertains to a particular customer, I need to see that information exactly rather than a recommended event.

**Comment by jbccollins** [comment]
This was an incredibly confusing experience for a new Sentry developer. I did not understand why test events I was firing were not showing up the way I expected.

**Comment by fmichaut-diff** [comment]
Also changed my default to "latest event". On a fast moving project, it's always more informative to look at the trace of the latest event since the code might have changed in the meantime.
